tests/bluetooth/qualification/ICS_Zephyr_Bluetooth_Host.pts

```diff
@@ -29,14 +37,26 @@
         <table>14a</table>
         <row>7</row>
       </item>
+      <item>
+        <table>23</table>
+        <row>7a</row>
+      </item>
       <item>
         <table>27b</table>
         <row>7</row>
       </item>
+      <item>
+        <table>25</table>
+        <row>13b</row>
+      </item>
       <item>
         <table>37c</table>
         <row>2</row>
       </item>
+      <item>
+        <table>37</table>
+        <row>3a</row>
+      </item>

```

Comment:
What about 37/4? 
![image](https://github.com/user-attachments/assets/1908aa35-58a2-434f-8773-9b83bac73136)

Reply:
there is no support for " LE GATT Security Levels Characteristic"

Reply:
Ah, so it's a characteristic we don't have in Zephyr?

Reply:
yeap, although with dynamic gatt database it may not be that trivial to implement (unless we left this up to application...)